entrypoint.sh

```diff
@@ -6,16 +6,15 @@ if [ "${#}" -eq 0 ]; then
     exit 1
 fi
 
-if tty --silent; then
```

Comment:
could you elaborate why this should be obsolete now? 

From what I remember this was useful for `heroku run` situations

Reply:
as i worked with it, it tended to be less predictable and more surprising. for example, `docker compose run my_service` attaches a tty, which prevents `my_service` from actually running its service with the firewall.

in other words, `tty --silent` is expected behavior if you tack a `/bin/bash` to the end of a `docker compose run` command, but surprising behavior otherwise. better to have an environment variable that you can switch on and off intentionally.

also i discovered that `heroku run` has an `env` parameter, so you can run `heroku run --env SIGSCI_STATUS=disabled /bin/bash` and you should be able to debug. not ideal, but more predictable.